### DISCUSSION TITLE
Thank you very much for you work

### DISCUSSION THREAD

**Discussion by iurev** [neovide/neovide#499] [General]
Thank you for neovide, I really love it!
This is the editor I wanted so much, it's just a missing piece of the puzzle for me!
Animations are awesome, it's just a piece of inspiration for me!

**Comment by Kethku** [comment]
Comments like this are what keep me working on it. Thank you so much.

**Comment by Kethku** [comment]
I converted this to a discussion for selfish reasons so that I can keep it around ;)

**Comment by iurev** [comment]
By the way, @Kethku is there any way to donate?

**Reply by Kethku** [reply]
I just setup a Kofi page. Thanks for the kick! https://ko-fi.com/kaysimmons

I'm going to add it to the readme.

**Reply by iurev** [reply]
Thank you, done :)

**Comment by alannotnerd** [comment]
I found we just upload the artifacts but don't create release automatically.  I wanna to make a pr for this if we need? @Kethku

**Reply by Kethku** [reply]
Absolutely! A PR for this would be awesome. Something which creates releases from main branch builds as develop releases or similar would be awesome

**Comment by chrisbouchard** [comment]
I agree with this 100%. Thank you very much for this awesome tool.

**Comment by mightyiam** [comment]
I would like to thank you, as well, @Kethku. I have just installed Neovide, ran it for the first time and couldn't believe it. I'm running vim with animations. Something about this blows my mind. I burst into maniacal laughter. I'm pretty sure my wife heard. And that this is written in Rust is the cherry on the cream. I'm proud of you and grateful for your work.

**Comment by randywallace** [comment]
Thank you again for this app!
